### PR TITLE
Fix example in endpoint

### PR DESCRIPTION
**Pull request by subatoi**
Provides actual values

### CODE REVIEW

File: source/documentation/using_the_api/api_reference.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ Parameters:
 * `limit` (Optional): Collection page size. Defaults to 100. Maximum is 5000. 
 
 ```http
-GET /entries/?start=1&limit=10 HTTP/1.1
+GET /entries?start=1&limit=10/ HTTP/1.1
```
Comment (gidsg) [inline source/documentation/using_the_api/api_reference.md]: trailing slash here is incorrect.

**Review by gidsg — CHANGES_REQUESTED** [review]

File: source/documentation/using_the_api/api_reference.md [hunk 2]
```diff
@@ -272,7 +272,7 @@ Authorization: YOUR-API-KEY-HERE
 ```http
 HTTP/1.1 200
 Content-Type: application/json
-Link: <?start=101&limit=100>; rel="next"
+Link: <?start=1&limit=10>; rel="next"
```
Comment (arnau) [inline source/documentation/using_the_api/api_reference.md]: The response for `start=1&limit=10` is `Link: <?start=11&limit=10>; rel="next"`.

You can check the actual values for a response with curl with the flag `-I` (capital i):

```
curl -I "https://country.register.gov.uk/entries?start=1&limit=10"
```

**Review by arnau — CHANGES_REQUESTED** [review]

File: source/documentation/using_the_api/api_reference.md [hunk 3]
```diff
@@ -263,7 +263,7 @@ Parameters:
 * `limit` (Optional): Collection page size. Defaults to 100. Maximum is 5000. 
 
 ```http
-GET /entries/ HTTP/1.1
+GET /entries?start=1&limit=10 HTTP/1.1
```
Comment (arnau) [inline source/documentation/using_the_api/api_reference.md]: I suspect this should be `/entries/?start=1&limit=10` to keep the style consistent?

**Review by arnau — APPROVED** [review]

**Comment by bahmady** [comment]
If no objections I'll merge this

**Pull request merged**